components/02-elements/list/_list.scss

```diff
@@ -32,6 +32,22 @@ $list-divider            : 1px solid $gray-dark !default;
         }
     }
 
+    &__label {
+        display: inline-block;
+        clear: left;
```

Comment:
`inline-block` with `float` makes no sense at all and liters will yell at you
Also we should avoid using `float`s and `inline-block`s at all.